docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md

```diff
@@ -0,0 +1,563 @@
+
+- **Hardware Requirements**
+
+    Your machine type must meet the requirements defined in the [prerequisites](../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation.md#prerequisites).
+
+- **Worker Nodes**
+
+    The number of worker nodes on which IO engine pods are deployed should not be less than the desired replication factor when using the synchronous replication feature (N-way mirroring).
+
+- **Additional Disks**
+
+    Your worker nodes should have additional storage disks attached. The additional storage disks should not be mounted or contain a filesystem.
+
+- **Enable Huge Pages**
+    
+    2MiB-sized Huge Pages must be supported and enabled on the storage nodes i.e. nodes where IO engine pods are deployed. A minimum number of 1024 such pages (i.e. 2GiB total) must be available exclusively to the IO engine pod on each node.
+    Huge pages in the OpenShift Container Platform (OCP) can be enabled during the installation or it can be enabled by creating new machine configs post-installation. Refer to the [Red Hat Documentation](https://access.redhat.com/solutions/5214791) for more details.
+
+- **Kernel Modules**
+
+    nvme modules are loaded by default in coreOS.
+
+- **Preparing the Cluster**
+
+    Refer to the [Replicated PV Mayastor Installation Documentation](../../user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-installation.md#preparing-the-cluster) for instructions on preparing the cluster.
+
+- **Security Context Constraint (SCC)**
+
+    Ensure that the service account used for the OpenEBS deployments is added to the privileged SCC.
+
+    ```
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-promtail
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-loki
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-localpv-provisioner
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-nats
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-lvm-controller-sa
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-lvm-node-sa
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-promtail
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-service-account
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-zfs-controller-sa
+    oc adm policy -n openebs add-scc-to-user privileged -z openebs-zfs-node-sa
+    oc adm policy -n openebs add-scc-to-user privileged -z default
+    ```
+
+## Install Replicated PV Mayastor on OpenShift
+
+Refer to the [OpenEBS Installation Documentation](../../quickstart-guide/installation.md#installation-via-helm) to install Replicated PV Mayastor using Helm.
+
+- **Helm Install Command**
+
+```
+helm install openebs --namespace openebs openebs/openebs --create-namespace --set openebs-crds.csi.volumeSnapshots.enabled=false 
```

Comment:
This is installing the umbrella and not just replicated

Reply:
Yes, but we haven't really exposed the options to disable lvm/zfs.

Reply:
We have: https://github.com/openebs/openebs/blob/065d171cd3ac13ac334879600cc69ce4388889b1/charts/values.yaml#L56

Reply:
Yes, we have. I'm saying we have not consciously exposed these options.